docs/ref/settings.txt

```diff
@@ -318,8 +318,9 @@ Default: ``None``
 The domain to be used when setting the CSRF cookie.  This can be useful for
 easily allowing cross-subdomain requests to be excluded from the normal cross
 site request forgery protection.  It should be set to a string such as
-``"example.com"`` to allow a POST request from a form on one subdomain to be
-accepted by a view served from another subdomain.
+``".example.com"`` to allow a POST request from a form on one subdomain to be
+accepted by a view served from another subdomain. See :ref:`how-csrf-works` for
```

Comment:
I'm not sure if the link adds sufficient value considering the "details" are five bullet points down. (And if you go with my suggested simpler change, I don't think there's much additional information.)